test/blackbox/common/mesh_event_handler.h

```diff
@@ -90,6 +90,20 @@ struct cond_flag {
 	bool flag;
 };
 
+typedef struct {
+	mesh_event_t  currentEv;
+	mesh_event_t  nextEv;
+} meshlink_node_status_lookup_t;
+
+typedef struct {
+	uint16_t                        node_id;
+	mesh_event_t                    currentEv;
+	meshlink_node_status_lookup_t   *lookup;
+	uint16_t                        current_index;
```

Comment:
Why is this one `uint16_t` and `max_events` only `uint8_t`? Also, this is not performance critical in any way, so I would just use `int` here for both.

Reply:
I have changed both of them to int